workflow/rules/functions.smk

```diff
@@ -380,6 +386,10 @@ def build_filename_constraints(which):
 def select_container_readme(wildcards):
 
     _, refcon_md = _get_container_metadata(wildcards.rc_name_version)[0]
+    if 'base' in {wildcards.rc_name_version}:
+        base_container_path = pl.Path(f'container/{wildcards.rc_name_version}.sif')
+        if not base_container_path.is_file():
+            raise ValueError('No base container found. Please follow instructions above to create base container')
```

Comment:
please make this even more explicit:
"...instructions above (see: ACTION REQUIRED) to create..."